.travis.yml

```diff
@@ -18,12 +14,9 @@ before_install:
     - mv ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini{,.disabled} || echo "xdebug not available"
     - travis_retry composer self-update
 
-install:
-    - composer require react/promise:2.*
-    - composer require psr/http-message:1.*
-    - travis_retry composer update --prefer-dist
+install: composer install --dev --prefer-dist
 
-script: ./vendor/bin/phpunit --group default,ReactPromise
+script: bin/phpunit --group default,ReactPromise
```

Comment:
as mentioned in https://github.com/webonyx/graphql-php/pull/333/files#r213260516, this should be reverted